.github/workflows/api-baseline.yml

```
@@ -0,0 +1,61 @@
+          cache: npm
+
+      - run: npm ci
+      - run: npm run build
+      - run: npm run contract:baseline
+
+      # A GITHUB_TOKEN push does not itself start another workflow run, which is
+      # fine here: the regenerated baseline is byte-identical to what the
+      # `api-contract` check would compare against, so its verdict on the PR is
+      # unchanged, and the Release PR is maintainer-merged regardless.
+      - name: Commit the regenerated baseline
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)